src/lib/utils/meta_tables/meta_tables_table.hpp

```diff
@@ -15,7 +15,7 @@ class MetaTablesTable : public AbstractMetaTable {
  protected:
   friend class MetaTableManager;
 
-  std::shared_ptr<Table> _on_generate() const final;
+  std::shared_ptr<Table> _on_generate() final;
```

Comment:
Where did the removal of const become necessary? I must have missed something in the diff. Generally, I worry that a non-const generate method leads to state being stored in the table and thus, the generations not being independent.

Reply:
I added a count for the generate method of the mock table to verify it is not generated multiple times.

Reply:
So the non-constness is just needed in the test? In that case, simply mark the member as mutable.